nodejs-example-logs-api-extension/nodejs-example-logs-api-extension/extensions-api.js

```diff
@@ -0,0 +1,48 @@
+const fetch = require('node-fetch');
+const {basename} = require('path');
+
+const baseUrl = `http://${process.env.AWS_LAMBDA_RUNTIME_API}/2020-01-01/extension`;
+
+async function register() {
+    const res = await fetch(`${baseUrl}/register`, {
+        method: 'post',
+        body: JSON.stringify({
+            'events': [
+                'INVOKE',
+                'SHUTDOWN'
+            ],
+        }),
+        headers: {
+            'Content-Type': 'application/json',
+            'Lambda-Extension-Name': basename(__dirname),
```

Comment:
Worth adding a quick note on top of this that the name of the extension must match the file name of the extension itself that's in `/opt/extensions/`, that is `nodejs-example-logs-api-extension`.